src/azul/indexer/__init__.py

```diff
@@ -352,6 +352,17 @@ def __len__(self):
         """
         return len(self.common) + self.partition
 
+    def __contains__(self, partition: str) -> bool:
+        """
+        Efficiently compute whether the partition is a member of
+        :meth:`partition_prefixes`
```

Comment:
```suggestion
        Same as `partition_prefix in prefix.partition_prefixes()` but more efficient. See also :meth:`partition_prefixes`.
```

and add doctests, please.